indigo/distribution.yaml

```diff
@@ -780,6 +780,14 @@ repositories:
       url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
       version: master
     status: maintained
+  bowpmap_ros-release:
```

Comment:
I want my source and release repository to be same. When i release using bloom,  it asks for release url, so i have given the same source url to it. How do i resolve it?